APIs/Eminer.ps1

```diff
@@ -22,9 +22,12 @@ class Eminer : Miner {
         }
 
         $HashRate_Name = [String]$this.Algorithm[0]
-        $HashRate_Value = [Double]($Data.devices.hashrate_1m | Measure-Object -Sum).Sum
+#        $HashRate_Value = [Double]($Data.devices.hashrate_1m | Measure-Object -Sum).Sum
+        $HashRate_Value = [Double]($Data.total_hashrate_mean | Measure-Object -Sum).Sum
 
-        $HashRate | Where-Object {$HashRate_Name} | Add-Member @{$HashRate_Name = [Int64]$HashRate_Value}
+        if ($HashRate_Name -and $HashRate_Value -gt 0) {
```

Comment:
skip 0 hash rate values for more accurate reading. Sometimes the first few reding report 0, just because the miner is not ready yet.
If all was 0 then the miner would be dropped anyway